src/PKSim.Presentation/Presenters/Parameters/TableParameterPresenter.cs

```diff
@@ -23,7 +23,7 @@ public interface ITableParameterPresenter : ICommandCollectorPresenter
       void Save();
       void AddPoint();
       TableFormula EditedFormula { get; }
-      IEnumerable<ValuePointDTO> AllPoints();
+      IEnumerable<ValuePointDTO> AllPoints { get; }
```

Comment:
I did a global text search/replace for AllPoints() => AllPoints and this one also got changed in the process. I suppose it is ok this way. LMK

Reply:
Loooks good to me